kqueen/auth/common.py

```diff
@@ -9,28 +9,71 @@
 import importlib
 import logging
 
+config = current_config()
 logger = logging.getLogger('kqueen_api')
 
+"""
+    Authentication Modules
+
+    To define new module, need to specify it as dictionary, where:
+    "auth_option_lower_case": {"engine": "EqualsToAuthClassName",
```

Comment:
I would still like this dict to be formatted properly, it just looks bad, sry :)

Reply:
ah, let me change)